docs/src/rules/no-constant-condition.md

```diff
@@ -118,14 +118,94 @@ if(input === "hello" || input === "bye"){
 
 ### checkLoops
 
-Set to `true` by default. Setting this option to `false` allows constant expressions in loops.
+This is an object option having following values:
 
-Examples of **correct** code for when `checkLoops` is `false`:
+* `all` - Disallow constant expressions in all loops.
+* `allExceptWhileTrue` (default) - Disallow constant expressions in all loops except `While` loop with expression `true`.
+* `none` - Allow constant expressions in loops.
+
+Or instead you can set the `checkLoops` value to booleans where `true` is same as `all` and `false` is same as `none`.
+
+Example of **incorrect** code for when `checkLoops` is `all` or `true`:
```

Comment:
We typically use double quotes for string values.

```suggestion
* `"all"` - Disallow constant expressions in all loops.
* `"allExceptWhileTrue"` (default) - Disallow constant expressions in all loops except `while` loops with expression `true`.
* `"none"` - Allow constant expressions in loops.

Or instead you can set the `checkLoops` value to booleans where `true` is same as `"all"` and `false` is same as `"none"`.

Examples of **incorrect** code for when `checkLoops` is `"all"` or `true`:
```

Reply:
Done!